specs/dsl.md

```diff
@@ -0,0 +1,150 @@
+# DSL 
+DSL is the *domain specific language* used to communicate with a single zero-os node. The language is designed to 
```

Comment:
We need to rephrase this. There will be no communication to the node itself.
Instead provisioning will be driven from transactions in a blockchain.
The DSL will still be used as a description of what needs to be provision, but the node will just read it from the blockchain. So there is not really an API to create resources.